### ISSUE TITLE
uninstalling yarn

### ISSUE THREAD

**Issue by dantejauregui** [yarnpkg/website#514]
Hello guys!

if I install through this way:   "curl -o- -L https://yarnpkg.com/install.sh | bash"

how can i uninstall correctly??  please :D

**Comment by Daniel15** [comment]
All it does is extracts the tarball to `~/.yarn` so you can simply delete that directory. Maybe in the future we could have an uninstall script.

**Comment by jaydenseric** [comment]
@Daniel15 Surely the install script modified profiles, etc.?

Ideally there would be both an uninstall script, and manual removal instructions on the website. Currently I'm attempting to understand the install script line-by-line.

**Comment by Daniel15** [comment]
Oh yeah, it also adds `~/.yarn/bin` to the PATH. Feel free to submit a pull request adding an uninstallation script if you like.

**Comment by AlexKvazos** [comment]
Spent so much time looking for where the install script added the PATH set and never found it so I have to keep the .yarn/bin folder now...

**Comment by RKushnir** [comment]
@AlexKvazos For me it was in `~/.zshrc`, check the respective file for your shell.

**Comment by metareven** [comment]
Why is this issue closed?

**Comment by Haroenv** [comment]
We should be able to have an uninstall script which also detects the profile and deletes it

https://github.com/yarnpkg/website/blob/67843c54fcf5625284081d35f3437523bb4da5fd/install.sh#L115-L155

**Comment by MichelDiz** [comment]
Precisely. At this point I am trying to clean everything I have from Yarn to install it again. Because I'm having "network problems". I want to "zero" and start from scratch. To be sure.

I think I installed it on my Mac Via Sh. I've tried all ways to uninstall and nothing. I have to find out.

**Comment by jle-pass** [comment]
+1
I pity those who wait ~~six~~ seven years to have a functionality that should be present from the design of any software :'-(
